### PR TITLE
Admin - manage user's projects

### PR DESCRIPTION
**Pull request by AntonMajest**

### CODE REVIEW

File: web/src/components/admin/ContractsList/ContractsList.tsx [hunk 1]
```diff
@@ -110,15 +111,19 @@ const ContractsList: React.FC<Props> = ({ users, projects, statuses }) => {
...
+                  moment(DT.now().toISODate()).format('YYYY-MM-DD') || !user.inactive && contract?.end_date === null ? (
                     <TableRow key={contract.id} 
                       className={contract ? backgroundColorUser(contract.job_status_id, user.inactive) : ''}>
                       <TableCell align="center" sx={tableCellStyles}>
                         {numbering++}
                       </TableCell>
+                      {console.log(moment(contract?.end_date).format('DD-MM-YYYY') < 
+                    moment(DT.now().toISODate()).format('DD-MM-YYYY'))}
+                      {console.log(contract?.end_date)}
+                      {console.log(moment(DT.now().toISODate()).format('YYYY-MM-DD'),'NOW')}
```
Comment (djmaluy) [inline web/src/components/admin/ContractsList/ContractsList.tsx]: console.logs

File: web/src/components/admin/ContractsList/ContractsList.tsx [hunk 2]
```diff
@@ -110,15 +111,19 @@ const ContractsList: React.FC<Props> = ({ users, projects, statuses }) => {
...
+                return !user.inactive && contract?.end_date >
+                  moment(DT.now().toISODate()).format('YYYY-MM-DD') || !user.inactive && contract?.end_date === null ? (
                     <TableRow key={contract.id} 
                       className={contract ? backgroundColorUser(contract.job_status_id, user.inactive) : ''}>
                       <TableCell align="center" sx={tableCellStyles}>
                         {numbering++}
                       </TableCell>
+                      {console.log(moment(contract?.end_date).format('DD-MM-YYYY') < 
+                    moment(DT.now().toISODate()).format('DD-MM-YYYY'))}
+                      {console.log(contract?.end_date)}
```
Comment (djmaluy) [inline web/src/components/admin/ContractsList/ContractsList.tsx]: console.logs

**Review by djmaluy — COMMENTED** [review]
+1

File: web/src/components/admin/ContractsList/ContractsList.tsx (hunk 2 above)
Comment (AntonMajest) [reply web/src/components/admin/ContractsList/ContractsList.tsx]: resolved in next commit

**Review by NazarYanchyshyn — APPROVED** [review]
+1

**Pull request merged**